charts/s1-agent/values.yaml

```diff
@@ -80,6 +80,7 @@ helper:
   fullnameOverride: ""
   nameOverride: ""
   labels: {}
+  podAnnotations: ""
```

Comment:
```suggestion
  podAnnotations: {}
```

Reply:
@andrewjamesbrown I was going to complete it this way, however that isn't how the default is set for `podAnnotations` under `agent`.